### ISSUE TITLE
AI Code companion

### ISSUE THREAD

**Issue by ashik529** [vikash-bhardwaj/ai-code-companion-documentation#3]
Hi,I am facing this issue for 2 days .**API call failed with status code: 401 and error message: **Unauthorized.**

![Screenshot 2023-10-31 131203](https://github.com/vikash-bhardwaj/ai-code-companion-documentation/assets/149458880/427bc04a-2f82-4a92-bd83-e4fe44a2f9c4)

**Comment by ashik529** [comment]
I have installed AI code companion and set access token then access token has been saved.

**Comment by vikash-bhardwaj** [comment]
Hi @ashik529  - I am sorry for late response and I saw your message while checking my notifications email, I have already responded with the possible reason you facing 429 (Too many request). But mentioning here again in case others are also facing same issue - Error code 429 - Too many requests should occur when you work with OpenAI free account or have not set-up the billimg limit. I hope 401 was resolved by providing the right Access Token?

Let me know if you still facing any other issue @ashik529
